boards/common/iotlab/Makefile

```diff
@@ -1,3 +1,3 @@
-MODULE = arduino-mkr-common
```

Comment:
next time use "git mv" ;)

Reply:
Git does not handle renames. `git mv` just calls `git rm`, `mv` and `git add`. This is likely a glitch from gits magic rename detection code that tries to guess if files were renamed while looking at a changeset.